metis-harvesting/src/main/java/eu/europeana/metis/harvesting/oaipmh/OaiHarvesterImpl.java

```diff
@@ -81,10 +81,10 @@ private static ListIdentifiersParameters prepareListIdentifiersParameters(OaiHar
     ListIdentifiersParameters parameters = ListIdentifiersParameters.request()
             .withMetadataPrefix(harvest.getMetadataPrefix());
     if (harvest.getFrom() != null) {
-      parameters.withFrom(Date.from(harvest.getFrom()));
+      parameters.withFrom(Date.from(harvest.getFrom()).toInstant());
```

Comment:
harvest.getFrom() returns instant already, so the Date.from.toinstant is not required I believe.
Same below.

Reply:
oops. I see. thanks. done.